eng/.docsettings.yml

```diff
@@ -6,6 +6,12 @@ omitted_paths:
   - sdk/identity/identity/test/manual/*
   - sdk/identity/identity/test/manual-integration/*
   - sdk/keyvault/*/test/**
+  - sdk/appconfiguration/*/test/**
+  - sdk/eventhub/*/test/**
+  - sdk/search/*/test/**
+  - sdk/servicebus/*/test/**
+  - sdk/storage/*/test/**
+  - sdk/textanalytics/*/test/**
```

Comment:
Why the double ** in the end? We are trying to list the readme for the test folder right?

Reply:
Done!